src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -120,6 +120,25 @@ Example:
     import { Credentials } from '../../../shared/models/credentials.model';
     import { HostService } from './services/host.service';
 
+Frontend components
+~~~~~~~~~~~~~~~~~~~
+
+There are several components that can be reused on different pages.
+This components are declared on the components module:
+`src/pybind/mgr/dashboard/frontend/src/app/shared/components`.
+
+Helper
+~~~~~~
```

Comment:
This should be a sub-item of "Frontend components"

Reply:
Fixed